test/test_vim.py

```diff
@@ -22,7 +22,8 @@ def test_command():
     vim.command('w')
     ok(os.path.isfile(fname))
     eq(open(fname).read(), 'testing\npython\napi\n')
-    os.unlink(fname)
+    if os.name != 'nt':
```

Comment:
in the previous suggestion was for sending 'q', maybe that call to unlink is made before neovim actually shuts down. just for testing purposes try sleeping for a second before calling unlink just to be sure
-